roles/edpm_nova/tasks/configure.yml

```diff
@@ -118,3 +120,33 @@
     mode: '0600'
     owner: nova
     group: nova
+
+- name: get fact for tls_certs_enabled
+  set_fact: tls_certs_enabled="{{ lookup('env','tls_certs_enabled') }}"
```

Comment:
should we use lookup ('env','tls_certs_enabled', true) so that empty (unset) env considered false?

Reply:
ack -- presumably set to false if not set.  This will allow older systems to continue to work

Reply:
no longer applicable